README.md

```diff
@@ -1,22 +1,19 @@
-# RSEM
-
-RSEM is a software package for estimating gene and isoform expression levels from RNA-Seq data. It has support for multi-threaded computation, can run on single-end and paired-end data and may produce statistic values in support of it's expression analysis. For visualization, It can generate BAM and Wiggle files in both transcript-coordinate and genomic-coordinate. For visualization, user may use RSEM-made bam and wiggle files with UCSC Genome browser or IGV browser from Broad Institute. RSEM can also make transcript read depth plots in pdf format. For this workflow, only limited subset of RSEM toolkit is used. This workflow provides a building block for RNAseq data analysis pipelines.
-
-![rsem flowchart](docs/RSEM_specs.png)
```

Comment:
This is a diagram we want to show, need to bring this back. To make things easier this block should be in the description section of **rsem.wdl**. This way, it will not get removed once README.md is re-generated

Reply:
done!